src/constants.ts

```diff
@@ -581,6 +581,7 @@ export interface CommandUser {
   username: string;
   avatar: string | null;
   avatar_decoration: string | null;
+  global_name: string | null;
   discriminator: string;
```

Comment:
for further future proofing (once everyone gets unique usernames),
```suggestion
  discriminator?: string;
```

Reply:
Would rather match closer to the PR's documentation. Discriminators are not optional.